packages/editor-ui/src/views/TestDefinition/TestDefinitionRunDetailView.vue

```diff
@@ -292,25 +392,77 @@ onMounted(async () => {
 	height: 200px;
 }
 
+.scrollableSummary {
+	border: var(--border-width-base) var(--border-style-base) var(--color-foreground-base);
+	border-radius: 5px;
+	background-color: var(--color-background-xlight);
+}
+
 .cardGrid {
-	display: grid;
-	grid-template-columns: repeat(auto-fit, minmax(6rem, 1fr));
-	gap: var(--spacing-xs);
-	margin-bottom: var(--spacing-m);
+	// display: grid;
+	// grid-template-columns: repeat(auto-fit, minmax(6rem, 1fr));
+	// gap: var(--spacing-xs);
+	// margin-bottom: var(--spacing-m);
+	// display: flex;
+	// border: var(--border-width-base) var(--border-style-base) var(--color-foreground-base);
+	// border-radius: 6px;
+	// overflow-x: auto;
+	// scrollbar-color: var(--color-foreground-base) transparent;
+	// min-height: 94px;
+	// &::-webkit-scrollbar {
+	// 	height: 8px;
+	// }
+	// &::-webkit-scrollbar-thumb {
+	// 	background: rgba(0, 0, 0, 0.5);
+	// 	// border-radius: 4px;
+	// }
+	// &::-webkit-scrollbar-track {
+	// 	background: transparent;
+	// }
 }
 
-:global {
-	.new {
-		color: var(--color-info);
-	}
-	.running {
-		color: var(--color-warning);
-	}
-	.completed {
-		color: var(--color-success);
+.summaryCard {
+	padding: var(--spacing-s);
+	/*border-top: 0;
+	border-bottom: 0;
+	border-left: 0;
+	border-radius: 0;*/
+	border-right: var(--border-width-base) var(--border-style-base) var(--color-foreground-base);
+	flex-basis: 169px;
+	flex-shrink: 0;
+	// background-color: transparent;
+	max-width: 170px;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+
+	&:first-child {
+		border-top-left-radius: inherit;
+		border-bottom-left-radius: inherit;
 	}
-	.error {
-		color: var(--color-danger);
+}
+
+// :global {
+// 	.new {
+// 		color: var(--color-info);
+// 	}
+// 	.running {
+// 		color: var(--color-warning);
+// 	}
+// 	.completed {
+// 		color: var(--color-success);
+// 	}
+// 	.error {
+// 		color: var(--color-danger);
+// 	}
+// }
+:global(.el-scrollbar__bar) {
+	opacity: 1;
+}
+:global(.el-scrollbar__thumb) {
+	background-color: var(--color-foreground-base);
+	&:hover {
+		background-color: var(--color-foreground-dark);
 	}
 }
```

Comment:
This will change the scrollbars globally. Maybe we should place inside a div from within this component